docs/publications.md

```diff
@@ -0,0 +1,360 @@
+---
+id: publications
+title: Publications
+---
+
+
+## Recent
+
+
+1. Indiana University (Fox, Qiu, Crandall, von Laszewski), Rutgers
+   (Jha), Virginia Tech (Marathe), Kansas (Paden), Stony Brook (Wang),
+   Arizona State (Beckstein), Utah (Cheatham), "Datanet: CIF21 DIBBs:
+   Middleware and High Performance Analytics Libraries for Scalable Data
+   Science" presentation at "Advancing the state-of-the-art in
+   high-performance computing, communications and data analytics"
+   workshop at Pittsburgh PA May 17-18 2016.
+
+   -  Full version: http://dsc.soic.indiana.edu/presentations/DibbsSPIDAL-May17-2016\_v4.pptx
+   -  Smaller version:  http://dsc.soic.indiana.edu/presentations/DibbsSPIDAL-May17-2016\_v4S.pptx
+
+## Rutgers University
+
+
+1. Hadoop on HPC: Integrating Hadoop and Pilot-based Dynamic Resource
+   Management Andre Luckow, Ioannis Paraskevakos, George
+   Chantzialexiou, Shantenu Jha Workshop on High-Performance Big Data
+   Computing, 2016 [`link <http://arxiv.org/abs/1602.00345>`_] [`bib
+   <https://www.github.com/saga-project/radical.wp/raw/master/publications/bib/hadoop-on-hpc.bib>`_]
+   : hadoop-on-hpc `http://arxiv.org/abs/1602.00345
```

Comment:
These are restructured text links but the format is markdown – needs to be changed.